mycore-iview2/src/main/java/org/mycore/iview2/services/MCRTilingQueue.java

```diff
@@ -43,6 +43,7 @@
 import jakarta.persistence.TypedQuery;
 
 public class MCRTilingQueue extends AbstractQueue<MCRTileJob> implements Closeable {
+    @SuppressWarnings("PMD.LooseCoupling")
```

Comment:
looks like a lazy implemented singleton pattern
maybe we should change this to avoid those suppressed warnings

Reply:
The problem is, that MCRTilingQueue implements a Collection interface therefor the SuppressWarnings is necessary here